src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,71 @@
+package duke;
```

Comment:
I like the package system. Especially what you did with resources nad testing! However, perhaps consider having Task, Event, Todo, TaskList in one package; Ui code in another package, and a package for utilities containing the Storage, Parser etc. classes.